include/util/query_heap.hpp

```diff
@@ -158,6 +158,13 @@ class QueryHeap
         node_index[node] = index;
     }
 
+    void InsertVisited(NodeID node, Weight weight, const Data &data)
```

Comment:
`InsertVisited` inserts node only into data vector and omits insertion into heap. It is needed only for source nodes, because source nodes weights are not real weights that must be 0, but  weights from projected source phantoms to ends of edge-based nodes. The special `relaxSourceEdges` function uses these weight values to relax source nodes.